hypothesis-python/docs/details.rst

```diff
@@ -632,3 +632,30 @@ it creates.  For example:
     use it in any way outside of type hints.  The only supported way to
     construct objects of this type is to use the functions provided by the
     :mod:`hypothesis.strategies` module!
+
+
+.. _pytest-plugin:
+
+----------------------------
+The Hypothesis Pytest Plugin
+----------------------------
+
+Hypothesis includes a tiny plugin to improve integration with :pypi:`pytest`,
+which is activated by default (but does not affect other test runners).
+It aims to improve the integration between Hypothesis and Pytest by
+providing extra information and a convenient access to config options.
```

Comment:
nit: is the “a” needed? Feels a bit jarring to me – “and convenient access to config options” scans more naturally.